i18n/id.json

```diff
@@ -0,0 +1,262 @@
+  "I18N_LIBRARY_NO_EXPLORATION_FOR_QUERY": "Ups, sepertinya tidak ada eksplorasi yang sesuai dengan pencarian anda.",
+  "I18N_LIBRARY_CREATE_EXPLORATION_QUESTION": "Apa anda ingin membuatnya?",
+  "I18N_LIBRARY_COMMUNITY_EXPLORATIONS": "Terjun ke dalam eksplorasi karya komunitas:",
+  "I18N_LIBRARY_EDIT_EXPLORATION": "Ubah",
+  "I18N_LIBRARY_EDIT_THIS_EXPLORATION": "Ubah eksplorasi ini",
+  "I18N_LIBRARY_EDIT_EXPLORATION_AS_MODERATOR": "Ubah (sebagai moderator)",
+  "I18N_LIBRARY_LAST_UPDATED": "Terakhir diperbarui",
+  "I18N_LIBRARY_EXPLORATION_FEATURED": "Utama",
+  "I18N_LIBRARY_LOADING": "Memuat",
+  "I18N_LIBRARY_UPDATED": "TERPERBARUI",
+  "I18N_LIBRARY_NO_OBJECTIVE": "Tidak ada tujuan yang tercantum.",
+  "I18N_LIBRARY_RATINGS_TOOLTIP": "Nilai",
+  "I18N_LIBRARY_VIEWS_TOOLTIP": "Dilihat",
+  "I18N_LIBRARY_NO_EXPLORATION_GROUPS": "Tidak ada grup eksplorasi yang dapat ditampilkan.",
+  "I18N_LIBRARY_VIEW_ALL": "Lihat semua",
+  "I18N_DASHBOARD_MY_EXPLORATIONS": "Eksplorasiku",
+  "I18N_DASHBOARD_MY_COLLECTIONS": "Koleksiku",
+  "I18N_TOPNAV_DASHBOARD": "Dashboard Pencipta",
+  "I18N_TOPNAV_NOTIFICATIONS": "Notifikasi",
+  "I18N_TOPNAV_PREFERENCES": "Preferensi",
+  "I18N_TOPNAV_MODERATOR_PAGE": "Halaman Moderator",
+  "I18N_TOPNAV_ADMIN_PAGE": "Halaman Admin",
+  "I18N_TOPNAV_LOGOUT": "Keluar",
+  "I18N_TOPNAV_SIGN_IN": "Masuk",
+  "I18N_TOPNAV_ABOUT": "Tentang",
+  "I18N_TOPNAV_ABOUT_OPPIA": "Tentang Oppia",
+  "I18N_TOPNAV_TEACH_WITH_OPPIA": "Mengajar dengan Oppia",
+  "I18N_TOPNAV_CONTACT_US": "Hubungi Kami",
+  "I18N_TOPNAV_LIBRARY": "Perpustakaan",
+  "I18N_CREATE_EXPLORATION_CREATE": "Buat",
+  "I18N_CREATE_EXPLORATION_UPLOAD": "Unggah",
+  "I18N_CREATE_ACTIVITY_TITLE": "Buat Aktivitas",
+  "I18N_CREATE_ACTIVITY_QUESTION": "Apa yang ingin anda buat?",
+  "I18N_CREATE_EXPLORATION_TITLE": "Buat Eksplorasi",
+  "I18N_CREATE_EXPLORATION_QUESTION": "Apa anda ingin membuat eksplorasi?",
+  "I18N_CREATE_EXPLORATION": "Buat Eksplorasi",
+  "I18N_CREATE_COLLECTION": "Buat Koleksi",
+  "I18N_CREATE_NEITHER_THANKS": "Tidak keduanya, Terima Kasih",
+  "I18N_CREATE_YES_PLEASE": "Ya, tentu!",
+  "I18N_CREATE_NO_THANKS": "Tidak, Terima Kasih",
+  "I18N_SIGNUP_PAGE_TITLE": "Bergabung ke komunitas - Oppia",
+  "I18N_SIGNUP_REGISTRATION": "Registrasi",
+  "I18N_SIGNUP_COMPLETE_REGISTRATION": "Lengkapi Registrasi Anda",
+  "I18N_SIGNUP_USERNAME_EXPLANATION": "Username anda akan ditampilkan di sebelah kontribusi anda.",
+  "I18N_SIGNUP_ERROR_NO_USERNAME": "Masukkan username.",
+  "I18N_SIGNUP_ERROR_USERNAME_WITH_SPACES": "Pastikan username anda tidak mengandung spasi.",
+  "I18N_SIGNUP_ERROR_USERNAME_MORE_50_CHARS": "Username hanya boleh memiliki maksimal 50 karakter.",
+  "I18N_SIGNUP_ERROR_USERNAME_ONLY_ALPHANUM": "Username hanya boleh menggunakan karakter alfanumerik.",
+  "I18N_SIGNUP_ERROR_USERNAME_WITH_ADMIN": "Username yang mengandung 'admin' telah terpakai.",
+  "I18N_SIGNUP_ERROR_USERNAME_NOT_AVAILABLE": "Username ini tidak tersedia.",
```

Comment:
Do you think "telah diambil" (or terambil?) would satisfy the context better? I'm not sure; just wondering.